src/Actions/ManagesTags.php

```diff
@@ -59,17 +77,66 @@ public function createTag(array $data = [])
      * Find or create a tag.
      *
      * @param string $name
+     * @param string $tagType
+     * @param string $description
      *
      * @return Tag
      */
-    public function findOrCreateTag($name)
+    public function findOrCreateTag($name, $tagType = 'contact', $description = null)
     {
         $tag = $this->findTag($name);
 
         if ($tag instanceof Tag) {
             return $tag;
         }
 
-        return $this->createTag(['tag' => $name]);
+        return $this->createTag(['tag' => $name, 'tagType' => $tagType, 'description' => $description]);
+    }
+
+    /**
+     * Updates a tag.
+     *
+     * @param Tag|int|string $id
+     * @param string|null    $tag
+     * @param string|null    $tagType
+     * @param string|null    $description
+     *
+     * @return Tag|null
+     */
+    public function updateTag($id, $tag = null, $tagType = 'contact', $description = null)
+    {
+        $id = $this->getTagId($id);
```

Comment:
Could be removed when using the Tag object

Reply:
Same thoughts [as above](https://github.com/testmonitor/activecampaign/pull/17#discussion_r314461289) apply here